documentation/api/development.md

```diff
@@ -0,0 +1,72 @@
+# Development
+
+This application is dockerized. Take a look at [Dockerfile](/api/Dockerfile) to see how it works.
+
+A very simple [docker-compose.yml](/docker-compose.yml) has been included to support local development and deployment. Take a look at [docker-compose.yml](/docker-compose.yml) for more information.
```

Comment:
Nit: This is a bit redundant, but it's ok!

Reply:
Good catch, thanks.